src/JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -296,14 +297,18 @@ private function denormalizeRelation(string $resourceClass, string $attributeNam
     {
         if (is_string($value)) {
             try {
-                return $this->iriConverter->getItemFromIri($value);
+                return $this->iriConverter->getItemFromIri($value, true);
```

Comment:
Why not catching the `EntityNotFoundException` here too (in another `catch` block)?